desktop/src/app/components/widgets/changes-history-modal.html

```diff
@@ -0,0 +1,61 @@
+<div class="modal-header">
+    <h5  class="modal-title" i18n="@@widgets.historyModal.header">
+        Bearbeitungshistorie der <b>{{document.resource.identifier}}</b>
+    </h5>
+
+</div>
+
+<div id="changes-history-modal-body" class="modal-body" >
+    <div class="resource-creation-info-row">
+        <span class="resource-creation-info-label" i18n="@@resource.creator.name">Erstellt von: </span>
```

Comment:
The i18n translation IDs should be named in a more consistent way. Usually, the name of the module and component is used (e. g. in this case "@@widgets.changeHistoryModal.creation.name", "@@widgets.changeHistoryModal.creation.date" etc.).

Code style: The last space inside the span can be removed (also below).